sdk/python/kfp/_credentials.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Arrikto Inc.
```

Comment:
Please use the standard header: "Copyright 2021 The Kubeflow Authors"

Reply:
FYI, all Google LLC headers have been changed to "The Kubeflow Authors".
Not a blocker, if you can update existing Arrikto Inc headers also to "The Kubeflow Authors" in a separate PR, that would be even better.

Reply:
@chensun @Bobgy this is a great initiative and we plan on changing all the copyrights as well.

However I think something is missing from this effort: an `AUTHORS` file. I've also left a comment in the issue: https://github.com/kubeflow/pipelines/issues/5470#issuecomment-833791582

Is it ok if we move forward like this and change the headers after we resolve this issue? (Or we have this file somewhere and I've missed it?)

Reply:
SGTM, we can do it separately too